xgo.go

```diff
@@ -238,49 +238,53 @@ func compile(image string, config *ConfigFlags, flags *BuildFlags, folder string
 		// Resolve the repository import path from the file path
 		config.Repository = resolveImportPath(config.Repository)
 
+		// Determine if this is a module-based repository
+		var modFile = config.Repository + "/go.mod"
```

Comment:
A `go.mod` file can be in any subdirectory of the repo, not only in its root (as stated in karalabe/xgo#144: https://github.com/karalabe/xgo/pull/144#pullrequestreview-182686786)

Reply:
Ok, I guess if thats the case we can leave it like this and add it later if needed. Should be fairly easy to add, but I also do think most of the time the `go.mod` file is in the root of the repo.